### ISSUE TITLE
Webpack warns about not finding source map files

### ISSUE THREAD

**Issue by danielberndt** [FormidableLabs/victory#2346]
When updating to victory v36.5.1, webpack issues a lot of warnings (159 in my case) with these contents:

```
WARNING in ../node_modules/victory/es/index.js
Module Warning (from ../node_modules/react-scripts/node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '[...]/node_modules/victory/es/index.js.map' file: Error: ENOENT: no such file or directory, open '[...]/node_modules/victory/es/index.js.map'
```

I'm using create react app 5.0.1 for my app. I.e. it's based on webpack 5.73.0

This was likely introduced with #2307

**Comment by marlonschlosshauer** [comment]
I have the same issue on a fresh React (18.2) install with Victory v36.5.3. Downgrading didn't fix it either (tried v36.0). Also seems to brick React's hot-reloading.

**Comment by siulca** [comment]
Same issue here on Victory v36.5.2

**Comment by miriyas** [comment]
Same issue here on Victory v36.5.3

**Comment by Chee7ah** [comment]
➕

**Comment by gbspr** [comment]
Any update to this bug or workaround?

**Comment by scottrippey** [comment]
I am currently investigating, and working on a fix.  Thank you for reporting!

**Comment by scottrippey** [comment]
Fixed in `36.6.3`, thanks for reporting!
